pyproject.toml

```diff
@@ -14,7 +14,7 @@ authors = []
 dependencies = [
     "certifi>=2021.5.30",
     "importlib_metadata>=4.8.1",
-    "qiskit_serverless>=0.17.0",
```

Comment:
I think we should start pinning the serverless package. If we include a breaking change old versions will not work.